p11-kit/meson.build

```diff
@@ -158,6 +159,14 @@ executable('p11-kit',
            link_with: [libp11_kit, libp11_kit_internal],
            install: true)
 
+if get_option('test')
+  executable('p11-kit-list-profiles-testable',
```

Comment:
Is there any reason this needs to be specific to the list-profiles subcommand? I mean, can we just create a p11-kit-testable executable from the same source files as p11-kit?

Reply:
when I run the p11-kit-testable I get this error message:
`p11-kit: invalid config filename, will be ignored in the future: /home/zfridric/upstream/p11-kit/p11-kit/fixtures/system-modules/two.badname`

Reply:
yes, it does affect the test execution. At least it shows up in the testlog.txt